go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -99,6 +99,7 @@ func init() {
 	flag.DurationVar(&Config.HeartbeatInterval, "heartbeat_interval", DefaultQsConfig.HeartbeatInterval, "How frequently to read and write replication heartbeat.")
 
 	flag.BoolVar(&Config.EnforceStrictTransTables, "enforce_strict_trans_tables", DefaultQsConfig.EnforceStrictTransTables, "If true, vttablet requires MySQL to run with STRICT_TRANS_TABLES on. It is recommended to not turn this flag off. Otherwise MySQL may alter your supplied values before saving them to the database.")
+	flag.BoolVar(&Config.DisableConsolidator, "disable-consolidator", DefaultQsConfig.DisableConsolidator, "Passing this flag turns off the query consolidator.")
```

Comment:
Think it's better to have this as `EnableConsolidator` with default value of true.